client/components/data/query-activity-log/README.md

```diff
@@ -1,24 +1,47 @@
-Query Activity Log
-==================
-
-A query component for fetching activity log data.
-
-## Usage
-
-```
-<QueryActivityLog siteId={ siteId } />
-```
-
-This component renders nothing. It ensures that any activity logs for the given site are available in global state.
-
-
-## Props
-
-### `siteId`
-
-<table>
-	<tr><th>Type</th><td>Number</td></tr>
-	<tr><th>Required</th><td>Yes</td></tr>
-</table>
-
-Site to fetch activities for.
+Query Activity Log
+==================
+
+A query component for fetching activity log data.
+
+## Usage
+
+```
+<QueryActivityLog siteId={ siteId } />
+```
+
+This component renders nothing. It ensures that any activity logs for the given site are available in global state.
+
+
+## Props
+
+### `siteId`
+
+Type   | Required
+------ | --------
+Number | Yes
+
+Site to fetch activities for.
+
+### `dateEnd`
+
+Type   | Required
+------ | --------
+Number | No
+
+Unix millisecond timestamp to fetch activity on or before.
```

Comment:
Since all the props except for `siteId` are not required and have no default values, probably it would be useful to  describe what happens if you omit any of them. For example, no `dateStart` would mean any activity from the beginning of time (or when we started logging, if we have to be precise).